### ISSUE TITLE
Intial Design

### ISSUE THREAD

**Issue by vipulob** [vipulob/quotes#1]
To design basic stricture for the Website using Django. 

Prerequisite:
1] Python 2.7
2] Django
3] sqlite

**Comment by vipulob** [comment]
Database Model:
1] Quote(quote_text=String, average_rating=Integer)
2] User(username=String,password=String)
3] Rating(user_rating=Integer)

**Comment by biju-r** [comment]
I was thinking of using some plugin that uses google/facebook authentication instead of we storing credentials. Also, first maybe we can just create a Quote and rating table and, for now, let anonymous unlimited rating system.

**Comment by vipulob** [comment]
Yes. Eventually we need to add single sign on. But for time being, untill we are confident we need to rely on Django's default User Authentication. Without User authentication we cannot portray rating system. So lets keep user Authentication too.  Thoughts?

**Comment by vipulob** [comment]
I have experimented little with Django's User Authentication. Has facility to create form's automatically. We should use it for being.

**Comment by biju-r** [comment]
that's great
